surface/inventory/models.py

```diff
@@ -12,3 +13,72 @@ class Application(models.Model):
     director = models.ForeignKey('Person', blank=True, null=True, on_delete=models.SET_NULL, related_name='+')
     director_direct = models.ForeignKey('Person', blank=True, null=True, on_delete=models.SET_NULL, related_name='+')
     dev_lead = models.ForeignKey('Person', blank=True, null=True, on_delete=models.SET_NULL, related_name='+')
+
+
+class FindingInheritanceQS(models.QuerySet):
```

Comment:
@fopina you comfortable with this query? We once discussed this might not be optimised, not sure what was the conclusion.

Reply:
Not sure about optimisation though one issue is that the QS is not used when it is a related field which is annoying (but not a real issue).

Apart from that, LGTM

Reply:
The intention here was to reduce the number of queries as much as possible. So, the relevant bit is actually the `get_children` method below. I'm not sure what else we could do for further optimisation.
We have been using it as is for considerably large models and it has been working great. Just wanted to double check if there is a better way :)